### DISCUSSION TITLE
How to change the root password?

### DISCUSSION THREAD

**Discussion by feb-ri** [Arkady23/V380-Web#49] [Q&A]
I managed to install the web interface and access the shell via Telnet. But when I tried to change the root password I learned that the file system is read-only. In this case, I have two questions:

1. If I change the value in `quick_check.ini` file prior to installation/setup, will that become my root password?
2. Is it possible to change the root password in read-only file system?

Thank you.

**Comment by Arkady23** [comment]
There's something about it [here](https://blog.caller.xyz/v380-ipcam-firmware-patching/).
